src/OpenLoco/src/World/Company.h

```diff
@@ -89,24 +89,92 @@ namespace OpenLoco
 
     constexpr size_t expenditureHistoryCapacity = 16;
 
+    enum class AiThinkState : uint8_t
```

Comment:
I wonder if we should move these AI related enums and structs into CompanyAi.h instead — even if they are used in Company.h as well. My thinking is this makes the Company.h less convoluted, thereby making it easier to find things, still.

Reply:
@Duncan, is this something you could pick up, or would you rather this be done in another PR?

Reply:
I was going to address it in this PR.